source/style-guide/style/code-examples.txt

```diff
@@ -10,14 +10,27 @@ Code Examples
    :depth: 1
    :class: singlecol
 
-Use the following guidelines when creating blocks of code as input
-or output examples:
+Use the following guidelines when creating code examples:
 
 -  Don't use screenshots to show code examples. Format them as blocks
    of code by using the appropriate markup in your authoring tool. For
```

Comment:
```suggestion
   of code by using the appropriate markup in your authoring tool, such as  `.. code-block::`. For
```
I believe `.. code-block::` is the main directive for creating code blocks if we want to mention explicitly!

Reply:
@lindseymoore, I'd like to keep this as in since we also have  ``.. io-code-block::``.